editor/src/clj/editor/gui.clj

```diff
@@ -1254,10 +1253,11 @@
 
 (defn- validate-material-capabilities [_node-id material-infos material material-shader texture-infos texture]
   (let [is-paged-material (shader/is-using-array-samplers? material-shader)
-        material-info (or (material-infos material)
-                          (material-infos ""))
+        material-info (or (get material-infos material)
+                          (get material-infos ""))
```

Comment:
`material-infos` can't actually be `nil` here, because you will always have the default material. I guess I just wanted to consistently use `get`, since the lookups come from optional keys in `basic-gui-scene-info`.